src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -94,6 +94,12 @@ module sonic-port{
 						pattern "on|off";
 					}
 				}
+
+				leaf autoneg {
+					type uint8 {
+						range 0..1;
```

Comment:
can you change this to on|off?

Reply:
how about the backward compatibility? we use 0 and 1 in the portsorch.cpp (swss container) today, after making on/off changes, hope 0 and 1 will also continue to work for autoneg, @dmytroxshevchuk please confirm.

Reply:
As per the below code, we need to change the backend first.
```
  /* Set autoneg and ignore the port speed setting */
                else if (fvField(i) == "autoneg")
                {
                    an = (int)stoul(fvValue(i));
                }
```